docs/book/content/schema/schemas_and_mutations.md

```diff
@@ -1,12 +1,13 @@
 # Schemas
 
-A schema consists of two types: a query object and a mutation object (Juniper
-does not support subscriptions yet). These two define the root query fields
-and mutations of the schema, respectively.
+A schema consists of three types: a query object, a mutation object and a subscription object
+( The usage of subscriptions is a little different from the mutation and query objects, so there is a
+specific [section][section] to handle this topic). 
+These three define the root query fields, mutations and subscriptions of the schema, respectively.
```

Comment:
I'd tell users to use `EmptySubscription` if they do not need the subscription functionality. I think it would help people to avoid creating empty subscription type from scratch:

```
If mutations/subscriptions functionality is not needed, consider using 
[EmptyMutation][EmptyMutation]/[EmptySubscription][EmptySubscription].
```